### DISCUSSION TITLE
Update needed.

### DISCUSSION THREAD

**Discussion by Rootkage** [nextcloud/all-in-one#2318] [Questions] [answered]
### Steps to reproduce
1. I make App updates via the normal app update.
2. After that, there come this picture: 
![image](https://user-images.githubusercontent.com/41064191/230917588-e762f93a-140e-44b3-9abc-12ef0d36e79f.png)
3.A restart doesen't help
I cannot figure out how to use the update command in my newest nexcloud aio setup.

### Expected behavior 
It has to be normal update the apps. Nothing more
### Actual behavior 
The Picture on the top for App updates

### Host OS 
Ubuntu 22.04 LTS

#### Nextcloud AIO version 
25.0.4 I think. Cannot look at it acutal
#### Other valuable info  

Could somebody tell me, how to make a update via CLI in the AIO Container? How is there something other to do?

Thanks :)

**Comment by Rootkage** [comment]
AIO 4.7.0

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/1787
